src/main/java/edu/harvard/iq/dataverse/api/auth/SignedUrlAuthMechanism.java

```diff
@@ -72,6 +75,18 @@ private User getAuthenticatedUserFromSignedUrl(ContainerRequestContext container
         }
         if (targetUser != null && userApiToken != null) {
             String signedUrl = URLDecoder.decode(uriInfo.getRequestUri().toString(), StandardCharsets.UTF_8);
```

Comment:
IIRC the servlet container preserves the (client's) case in getRequestUri() so this might not startWith("http://") unless it's lowered.